packages/suite-desktop/src-electron/libs/processes/BaseProcess.ts

```diff
@@ -135,6 +135,7 @@ abstract class BaseProcess {
         this.process = spawn(processPath, params, {
             cwd: processDir,
             env: processEnv,
+            stdio: ['ignore', 'ignore', 'ignore'],
```

Comment:
'ignore': equivalent to ['ignore', 'ignore', 'ignore'], but never mind...
```suggestion
            stdio: 'ignore',
```